package.json

```diff
@@ -1,18 +1,21 @@
 {
   "name": "gpv2poc",
   "license": "UNLICENSED",
+  "version": "0.0.1",
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
     "deploy": "node src/scripts/deploy.js",
-    "deploy:mainnet": "cross-env NETWORK=mainnet SUBGRAPH=gnosis/cow yarn deploy",
-    "deploy:rinkeby": "cross-env NETWORK=rinkeby SUBGRAPH=gnosis/cow-rinkeby yarn deploy",
-    "deploy:gc": "cross-env NETWORK=gc SUBGRAPH=gnosis/cow-gc yarn deploy"
+    "deploy:all": "run-s deploy:mainnet deploy:goerli deploy:gc",
```

Comment:
What does `run-s` do? Where does it come from? (Where does it live? That, and much more, tonight at...)

Reply:
That is coming from `yarn-run-all` package. Short for run synchronously.